change/@azure-msal-common-2021-01-28-23-58-32-api-extractor.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "none",
```

Assuming the exports are renamed: 

```suggestion
  "type": "major",
```